doc/overview.md

```diff
@@ -11,23 +14,17 @@ section {
 }
 </style>
 
-![bg left:40% 80%](img/rolling.png)
+![bg right:40% 80%](img/rolling.png)
 
-
-# ROS2 Load Balancing Service
+# [ROS 2 Load Balancing Service](https://github.com/Barry-Xu-2018/ros2_load_balancing_service)
 
 ---
 
-<style scoped>
-section {
-    font-size: 25px;
-}
-</style>
-
 # Objective
 
-- Support multiple service servers on the same service path to implement redundancy and load-balancing.
-- Existing ROS2 service server/client programs can be used without code modification.
+- ROS 2 service load-balancing in application layer without protocol change.
```

Comment:
there needs to be a space between `ROS` and `2` as official documentation. `ROS 2`, not `ROS2`...